platform/mikro-e/net-init.c

```diff
@@ -48,6 +50,44 @@
 #define PRINTF(...)
 #endif
 
+#define NVS_I2C_FREQUENCY   (300000)
```

Comment:
10kohm pull up on SCL and SDA. According to datasheet, this is the typical value of pull-up resistor when speed is 100kHz.